app/models/library_location.rb

```diff
@@ -2,12 +2,13 @@
 #  Class to handle configuration and logic around library codes and labels
 ###
 class LibraryLocation
-  attr_reader :library, :location
+  attr_reader :request, :library, :location
 
   include Scannable
   include Mediateable
 
   def initialize(request)
+    @request = request
```

Comment:
Just for clarity we decided that we would discuss the request coupling during future work and identify a consistent patter (so this shouldn't hold up this PR).